pkg/lib/indexer/indexer.go

```diff
@@ -79,6 +79,7 @@ func (i ImageIndexer) AddToIndex(request AddToIndexRequest) error {
 			}
 
 			databaseFile = path.Join(workingDir, dbLocation)
+			workingDir = path.Join(workingDir, path.Dir(dbLocation))
```

Comment:
I'm concerned about the `defer os.RemoveAll(workingDir)` but I wanted to get something up so I had time to think about test cases.